doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -784,3 +785,345 @@ JCL      - d247952 based on jdk-11.0.6+6)
+make -j$(nproc)
+make install
+```
+
+[2] Install `e2fsck` if it doesn't exist on your host system:
+```
+wget http://downloads.sourceforge.net/project/e2fsprogs/e2fsprogs/v1.43.1/e2fsprogs-1.43.1.tar.gz
+tar xzf e2fsprogs-1.43.1.tar.gz
+cd e2fsprogs-1.43.1
+./configure # <== If this step fails, please check the config.log file.
+                  It could be the missing "libc6-dev" package on your system.
+make
+make install
+```
+
+[3] Download the Fedora OS image `stage4-disk.img.xz` and `bbl` (the boot loader) from
+https://fedorapeople.org/groups/risc-v/disk-images/ (4GB in size by default)
+and expand it appropriately (e.g. 20GB) for more space as follows:
+```
+unxz --keep stage4-disk.img.xz
+truncate -s 20G stage4-disk.img
+e2fsck -fp stage4-disk.img
+resize2fs stage4-disk.img
+```
+
+[4] Boot the Fedora OS via QEMU (please refer to https://wiki.qemu.org/Documentation/Platforms/RISCV for details).
+e.g.
+```
+  qemu-system-riscv64 \
+   -nographic \
+   -machine virt \
+   -smp 4 \
+   -m 2G  \   # the memory can be adjusted depending on the capability of your host system
+   -kernel bbl \
+   -object rng-random,filename=/dev/urandom,id=rng0 \
+   -device virtio-rng-device,rng=rng0 \
+   -append "console=ttyS0 ro root=/dev/vda" \
+   -device virtio-blk-device,drive=hd0 \
+   -drive file=stage4-disk.img,format=raw,id=hd0 \
+   -device virtio-net-device,netdev=usernet \
+   -netdev user,id=usernet,hostfwd=tcp::10000-:22 (the port number is 10000)
+```
+Login: `root`
+Password: `riscv`
+
+The following screen messages show up after logging in the system with the root account
+e.g. 
+```
+Welcome to the Fedora/RISC-V stage4 disk image
+https://fedoraproject.org/wiki/Architectures/RISC-V
+
+Build date: 2018-04-06 08:24
+Kernel 4.19.0-rc8 on an riscv64 (ttyS0)
+The root password is 'riscv'.
+To install new packages use 'dnf install ...'
+If DNS isn't working, try editing '/etc/yum.repos.d/local.repo'.
+
+For updates and latest information read:
+https://fedorapeople.org/groups/risc-v/disk-images/readme.txt
+
+stage4 login: root
+Password: riscv
+Last login: Fri Oct  4 21:21:28 from 10.0.2.2
+
+[root@stage4 ~]# uname -a
+Linux stage4.fedoraproject.org 4.19.0-rc8 #1
+SMP Wed Oct 17 15:11:25 UTC 2018 riscv64 riscv64 riscv64 GNU/Linux
+```
+
+:bulb:
+To establish another session to communicate with the target system,
+run the command via any terminal software (e.g. putty) as follows:
+```
+  ssh -p 10000 root@localhost
+  root@localhost's password: riscv
+```
+
+[5] Install all X11/development related software packages required in building the JDK.
+```
+dnf install libX11-devel libXtst-devel libXt-devel libXrender-devel libXrandr-devel libXi-devel libXext-devel
+dnf install cups-devel fontconfig-devel alsa-lib-devel freetype-devel  #freetype might be skipped if already installed
+dnf install wget git autoconf automake  #mostly used in the compilation on Fedora/QEMU
+dnf install openssl-devel  #optional if the OpenSSL support is required)
```

Comment:
nit: please remove the closing parenthesis

Reply:
Removed.